.github/workflows/ci.yml

```diff
@@ -186,7 +186,6 @@ jobs:
 
       - name: Create Updated Documentation PR
         run: |
-          git fetch --depth=1 --no-tags
```

Comment:
yeah i wanted to optimize since fetching without this gets everything

Reply:
yeah, i dont think we need to fetch at all though, since it'll checkout documentation from origin, and i can merge straight from master